command/certificate/fingerprint.go

```diff
@@ -37,7 +42,14 @@ $ step certificate fingerprint /path/to/root_ca.crt
 Get the fingerprint for a remote certificate:
 '''
 $ step certificate fingerprint https://smallstep.com
-0d7d3834cf187726cf331c40a31aa7ef6b29ba4df601416c9788f6ee01058cf3
+e2c4f12edfc1816cc610755d32e6f45d5678ba21ecda1693bb5b246e3c48c03d
+'''
+
+Get the fingerprints for a remote certificate with its intemediate:
+'''
+$ step certificate fingerprint --bundle https://smallstep.com
+e2c4f12edfc1816cc610755d32e6f45d5678ba21ecda1693bb5b246e3c48c03d
```

Comment:
maybe something like 
```
0: e2c4f12edfc1816cc610755d32e6f45d5678ba21ecda1693bb5b246e3c48c03d
1: 25847d668eb4f04fdd40b12b6b0740c567da7d024308eb6c2c96fe41d9de218d
```
would make this more clear? nitpicky suggestion.

Reply:
Fixed with b3a4af9bf79a849cdd3dd17f2143dc522fa9e162